### ISSUE TITLE
Procedures with Attributes (Event / Internal) does not work

### ISSUE THREAD

**Issue by 365businessdev** [365businessdev/vscode-alxmldocumentation#1]
Events (Subscriber/Publisher) are not recognized/skipped when running cmds "Generate markdown documentation[...]"

Support for internal procedures would be nice :) At the moment it does not create documentation block ///

Original reported from [rating by Christian Ziepel](https://marketplace.visualstudio.com/items?itemName=365businessdevelopment.bdev-al-xml-doc&ssr=false#review-details)

**Comment by 365businessdev** [comment]
Sorry, we're unable to reproduce. If event is not local (see #2 ) Events (Internal, Business, Integration) and other Attributes (e.g. Obsolete) will be exported.

Used this code for testing:
``` javascript
/// <summary> 
/// Codeunit MyCodeunit2 (ID 50102).
/// </summary>
codeunit 50102 MyCodeunit2
{
    /// <summary> 
    /// Description for DoSomethingParam.
    /// </summary>
    /// <param name="param1">Parameter of type Text.</param>
    /// <param name="param2">Parameter of type Integer.</param>
    /// <returns>Return value of type Boolean.</returns>
    [Obsolete('This is an obsolete reason.')]
    procedure DoSomethingParam(param1: Text; param2: Integer): Boolean
    begin

    end;

    /// <summary> 
    /// Description for OnAfterCalcInvoiceSubscriber.
    /// </summary>
    /// <param name="SalesHeader">Parameter of type Record "Sales Header".</param>
    /// <param name="NewInvoice">Parameter of type Boolean.</param>
    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Sales-Post", 'OnAfterCalcInvoice', '', false, false)]
    procedure OnAfterCalcInvoiceSubscriber(SalesHeader: Record "Sales Header"; var NewInvoice: Boolean)
    begin

    end;
}
```

Could you pls provide example code to reproduce?

**Comment by MAC-Christian** [comment]
I found out that is not related to attributes. An underline in procedure name is causing this issue. We use an underline in Subscriber name to separate publisher name from subscriber task.

procedure GetHistory_OnAfterGetRecord()
begin
end;

**Comment by 365businessdev** [comment]
Ok, thank you!
We will check and fix in the next version.

**Comment by 365businessdev** [comment]
Fixed with #6 Version 0.3.0
